packages/augur-state/jest.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  "roots": [
+    "<rootDir>/src"
+  ],
+  "transform": {
+    "^.+\\.tsx?$": "ts-jest"
+  },
+  "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.tsx?$",
```

Comment:
In other projects we have been "co-locating" test files with a suffix. e.g. file.ts is tested in file.test.ts. I have no strong opinion either way. Just something to be aware of.

Reply:
I'm doing that here too. This file is just copy-pasted from an article on using typescript jest so it covers a few usecases. I can restrict it to just what we do though.